.github/actions/publish-nuget/New-NuGetPackage.ps1

```diff
@@ -32,5 +32,11 @@ foreach ($project in $projects)
         dotnet pack $project @Arguments
     }
 
+    if ($LASTEXITCODE -ne 0)
+    {
+        Write-Error "dotnet pack failed for the project $($project.FullName)."
```

Comment:
Shouldn't this be an error annotation?
```suggestion
        Write-Error "::error::dotnet pack failed for the project $($project.FullName)."
```

Reply:
Yep, new run: https://github.com/Lombiq/Orchard-Vue.js/actions/runs/4567982143 Note that I've used `Write-Output`, as we did in similar cases before. The result is that the "dotnet pack failed for the project /home/runner/work/Orchard-Vue.js/Orchard-Vue.js/Lombiq.VueJs/Lombiq.VueJs.csproj." error also appears under a run summary vs [not with `Write-Error`](https://github.com/Lombiq/Orchard-Vue.js/actions/runs/4567982143/attempts/5).